src/frontend/src/btc/components/core/BtcWalletAddress.svelte

```diff
@@ -0,0 +1,21 @@
+<script lang="ts">
+	import { isNetworkIdBTCMainnet } from '$icp/utils/ic-send.utils.js';
+	import Copy from '$lib/components/ui/Copy.svelte';
+	import { btcAddressMainnet, btcAddressTestnet } from '$lib/derived/address.derived';
+	import { networkId } from '$lib/derived/network.derived';
+	import { i18n } from '$lib/stores/i18n.store';
+	import { shortenWithMiddleEllipsis } from '$lib/utils/format.utils';
+
+	// Regtest and Testnet BTC wallets have the same address
+	const address = isNetworkIdBTCMainnet($networkId) ? btcAddressMainnet : btcAddressTestnet;
+</script>
+
+<div>
+	<label class="block text-sm font-bold" for="btc-wallet-address"
+		>{$i18n.wallet.text.wallet_address}:</label
+	>
+
+	<output id="btc-wallet-address" class="break-all"
+		>{shortenWithMiddleEllipsis({ text: $address ?? '' })}</output
```

Comment:
Using the default shortener with a default `splitLength` of 7 is also fine for bitcoin?

Reply:
Yes, as we can see in the receive modal where it's also implemented with the default `splitLength`, it works fine for BTC addresses:
<img width="407" alt="Screenshot 2024-09-25 at 13 10 23" src="https://github.com/user-attachments/assets/38742215-9c49-4ef5-a3cd-60a25b8827d7">